Cargo.toml

```diff
@@ -1,6 +1,9 @@
 [profile.release]
 panic = "unwind"
 
+[profile.production]
+inherits = "release"
```

Comment:
Why did we add this? Not saying it's bad, just curious

Reply:
ah, i was following how polkadot did theirs,

Reply:
Alright, no problem with that then